index.html

```diff
@@ -46,5 +42,189 @@ <h3 class="logo">John Doe </h3>
+    
+    <div class="tags">
+            <ul>
+                <li><a href="">html</a></li>
+                <li><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
+        </div>
+    
+        <button class="btn-2">See Project</button>
+    </div>
+
+    <div class="card">
+        <h4>Profesional Art Printing Data</h4>
+        <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. has been the industry's standard</p>
+    
+    <div class="tags">
+            <ul>
+                <li><a href="">html</a></li>
+                <li><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
+        </div>
+    
+        <button class="btn-2">See Project</button>
+    </div>
+
+    <div class="card">
+        <h4>Profesional Art Printing Data</h4>
+        <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. has been the industry's standard</p>
+    
+    <div class="tags">
+            <ul>
+                <li><a href="">html</a></li>
+                <li><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
+        </div>
+    
+        <button class="btn-2">See Project</button>
+    </div>
+
+    <div class="card">
+        <h4>Profesional Art Printing Data</h4>
+        <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. has been the industry's standard</p>
+    
+    <div class="tags">
+            <ul>
+                <li><a href="">html</a></li>
+                <li><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
+        </div>
+    
+        <button class="btn-2">See Project</button>
+    </div>
+
+    <div class="card">
+        <h4>Profesional Art Printing Data</h4>
+        <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. has been the industry's standard</p>
+    
+    <div class="tags">
+            <ul>
+                <li><a href="">html</a></li>
+                <li><a href="">bootstrap</a></li>
+                <li><a href="">Ruby</a></li>
+            </ul>
+        </div>
+    
+        <button class="btn-2">See Project</button>
+    </div>
+</div>
+
+<!--About Me Section-->
+<section class="abtSection">
+    
+
+    <section class="AboutMe">
+        <h1>About me</h1>
+        <p>Hello I’m a software developer! I can help 
+            you build a product , feature or website 
+            Look through some of my work and 
+            experience! If you like what you see and 
+            have a project you need coded, don’t 
+            hestiate to contact me.</p>
+            <a href=""><button>Get My Resume</button></a>  
+        <img src="images/abtFrame1.png" alt="About Image">    
+    </section>
```

Comment:
- I think it is preferable for your `About me` to be in an `<hx>` tag as well 👍🏽 

- I believe the `Get My Resume` can either be in a `<button>` or `<a>` anchor tag. I think you should kindly use one of them. For me, I would prefer 👉🏽 `<button>Get My Resume</button> `💯 

- It will be preferable if your `<img src="images/abtFrame1.png" alt="About Image">  ` is used as a background image in your CSS file instead because decorations are always backgrounded images, not `<img>` tags ✔️ 

- I guess you may have mistakenly added a closing section tag on line 186. Can you kindly remove it, I don't think it is necessary at this point 🤗